src/main/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiAction.java

```diff
@@ -73,8 +73,8 @@ public ActionGroupsApiAction(final Settings settings, final Path configPath, fin
 	}
 
 	@Override
-	public List<Route> routes() {
-		return routes;
+	public List<ReplacedRoute> replacedRoutes() {
+	    return replacedRoutes;
```

Comment:
mix of tabs and spaces for indentation.

Reply:
this line uses both spaces and tabs, it must use tabs for indentation.

Reply:
The same applies to several other files.

Reply:
fixed